src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -592,17 +592,13 @@ public function putAction(Request $request, $uuid)
         $language = $this->getLanguage($request);
         $type = $request->query->get('type', 'page');
 
-        try {
```

Comment:
Because we are using the [Exception Handling of the FOSRestBundle](http://symfony.com/doc/current/bundles/FOSRestBundle/4-exception-controller-support.html) now.